content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,95 @@
 
-This is also **the easiest way to integrate React into an existing website.** You can always add a larger toolchain if you find it helpful!
+Esta es también **la manera más fácil de integrar React en un sitio web existente.** Siempre puedes añadir una cadena de herramientas más grande si lo consideras útil!
 
-## Recommended Toolchains
+## Cadenas de Herramientas Recomendadas
 
-The React team primarily recommends these solutions:
+El equipo de React principalmente recomienda las siguientes soluciones:
 
-- If you're **learning React** or **creating a new [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app).
-- If you're building a **server-rendered website with Node.js,** try [Next.js](#nextjs).
-- If you're building a **static content-oriented website,** try [Gatsby](#gatsby).
-- If you're building a **component library** or **integrating with an existing codebase**, try [More Flexible Toolchains](#more-flexible-toolchains).
+- Si estás **aprendiendo React** o **creando una nueva [aplicación de página única](/docs/glossary.html#single-page-application),** usa [Create React App](#create-react-app).
+- Si estás construyendo un **sito web renderizado en servidor con Node.js,** prueba [Next.js](#nextjs).
+- Si estás construyendo un **sitio web orientado a contenido estático,** prueba [Gatsby](#gatsby).
+- Si estás construyendo una **biblioteca de componentes** o **integrando una base de código existente**, prueba [Cadenas de Herramientas más Flexibles](#cadenas-de-herramientas-ms-flexibles).
 
 ### Create React App
 
-[Create React App](http://github.com/facebookincubator/create-react-app) is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](/docs/glossary.html#single-page-application) application** in React.
+[Create React App](http://github.com/facebookincubator/create-react-app) es un ambiente cómodo para **aprender React**, y es la mejor manera de comenzar a construir **una nueva [aplicación de página única](/docs/glossary.html#single-page-application)** usando React.
 
-It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimizes your app for production. You’ll need to have Node >= 6 and npm >= 5.2 on your machine. To create a project, run:
+**Create React App** configura tu ambiente de desarrollo de forma que puedas usar las últimas características de Javascript, brindando una buena experiencia de desarrollo, y optimizando tu aplicación para producción. Necesitaras tener Node >= 6 y npm >= 5.2 instalados en tu máquina. Para crear un proyecto ejecuta:
```

Comment:
Maybe `Este` instead of `**Create React App**` could be closer to the original.

Also I suggest 

> ``Necesitarás tener Node >= 6 y npm >= 5.2 en tu máquina. Para crear un proyecto ejecuta:

Reply:
Yes, however I think that the explicit subject makes the sentence clearer, specially because the last noun used in the previous sentence was `React`.